lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -273,6 +273,63 @@ def _pixelsize(self):
         """Kymograph (spatial) pixel size in physical units."""
         return self._kymo.pixelsize[0]
 
+    def _model_fit(self, coordinate_idx):
+        """Extract fitted model from localization model at particular frame if available"""
+        num_coords = len(self)
+        coordinate_idx = num_coords + coordinate_idx if coordinate_idx < 0 else coordinate_idx
+        if not 0 <= coordinate_idx < num_coords:
+            raise IndexError(
+                f"Coordinate index is out of range of the KymoTrack. Kymotrack has length "
+                f"{num_coords}."
+            )
+
+        pixelsize, num_pixels = self._kymo.pixelsize[0], self._kymo.pixels_per_line
+        coords = np.arange(0.0, num_pixels * pixelsize, 0.1 * pixelsize)
```

Comment:
Good idea. I'll make a ticket on the backlog for it; since when doing this, it'd also be nice to change it in other places at the same time. I feel when adding the kymo cropping offset would be a good time to take this in that same sprint.